src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForSimulation.cs

```diff
@@ -214,6 +225,20 @@ private IMenuBarItem createConfigure(IMoBiSimulation simulation)
             .WithCommandFor<ConfigureSimulationUICommand, IMoBiSimulation>(simulation, _container);
       }
 
+      private IMenuBarItem createMakeSettingsProjectDefault(IMoBiSimulation simulation)
+      {
+         return CreateMenuButton.WithCaption(AppConstants.MenuNames.MakeSettingsProjectDefault)
+            // .WithIcon(ApplicationIcons.SimulationSettings)
+            .WithCommandFor<CommitSimulationSettingsUICommand, IMoBiSimulation>(simulation, _container);
+      }
+
+      private IMenuBarItem createRefreshSettings(IMoBiSimulation simulation)
+      {
+         return CreateMenuButton.WithCaption(AppConstants.MenuNames.RefreshSettingsFromProjectDefault)
+            // .WithIcon(ApplicationIcons.SimulationSettings)
```

Comment:
Here, and above, it makes sense to use this icon, but it's already used in the context menu. See the screenshot and let me know if it needs an icon, or can we go without one.